templates/messages.properties.j2

```diff
@@ -22,7 +22,7 @@
 screen.welcome.welcome=Congratulations on bringing CAS online!  To learn how to authenticate, please review the default authentication handler configuration.
 screen.welcome.security={{ cas_screen_welcome_security }}
 screen.welcome.instructions={{ cas_screen_welcome_instructions }}
-screen.welcome.label.netid=<span class="accesskey">U</span>sername:
+screen.welcome.label.netid=<span class="accesskey">O</span>UNetID (4x4):
```

Comment:
This is now a variable {{ cas_screen_label_netid }}. It is also in the main.yml file in vagrant_infrastructure